src/main/java/assignment/Decide.java

```diff
@@ -1,7 +1,7 @@
 package assignment;
 
 public class Decide {
-    public static boolean decide(Input input) {
+    static boolean decide(Input input) {
```

Comment:
Decide function is still not visible to the user outside our assignment package.